openfasoc/generators/ldo-gen/tools/compare_files.py

```diff
@@ -0,0 +1,38 @@
+def compare_files(template_file, result_file) -> int:
+    """Checks if the generated simulation result file matches with 
+    the stored template file.
+
+    Args:
+    - 'template_file' (string): The stored template file
+    - 'result_file' (string): The result file generated by the simulations
+    - 'errors' (dict): Dict of dicts containing max and min allowable errors (in percent) for each result type
+
+    Returns:
+    - 'int': 
+        Returns 2 if the differences in readings are greater than max allowable error,
+        Returns 1 if the differences in readings lie between the maximum and minimum allowable error range,
+        Returns 0 otherwise
+    """
+    errors = {
+        'frequency': { 'max': 1, 'min': 0.5 },
+        'power': { 'max': 1000, 'min': 1000 },
+        'error': { 'max': 100, 'min': 50 },
```

Comment:
Does the LDO have the same performance metrics as the other generators?

Reply:
No, so i'll keep this commented out until we have results for the ldo simulations